doc/api/errors.md

```diff
@@ -1338,10 +1338,17 @@ When using [`fs.cp()`][], `src` or `dest` pointed to an invalid path.
 
 <a id="ERR_FS_CP_FIFO_PIPE"></a>
 
-### `ERR_HTTP_CONTENT_LENGTH_MISMATCH`
+### `ERR_HTTP_BODY_NOT_ALLOWED`
```

Comment:
You swapped the description.

Reply:
I think the diff just looks strange. It's showing it as a deletion here, but I just moved line 1341 to 1347.